bindata/v3.11.0/kube-scheduler/installer-cluster-rolebinding.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: system:openshift:operator:cluster-kube-scheduler-installer
```

Comment:
the new installer code in library-code should create these for you :)

Reply:
https://github.com/openshift/library-go/pull/70

Reply:
i will update it today, just verifying that current PR works.